### ISSUE TITLE
train failed

### ISSUE THREAD

**Issue by codylcs** [embracesource-cv-com/FCHD#12]
E:\pycharmpro\mydetect\head\FCHD\networks\detector.py:30: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  probs = F.softmax(torch.from_numpy(rois_scores))
Fail to allocate bitmap
